database/src/main/resources/db/migration/V1_75__Add_function_for_associated_addresses.sql

```diff
@@ -0,0 +1,112 @@
+SELECT 'Add function get_tx_associated_values()' AS comment;
+
+drop function if exists get_tx_associated_values(text, integer);
+
+create or replace function get_tx_associated_values(tx_hash text, tx_height integer)
+    returns TABLE(value varchar, type varchar)
+    language plpgsql as
+$$
+BEGIN
+    RETURN QUERY
+-- BASE tx info
+        with tx AS (
+            select id from tx_cache where hash = tx_hash and height = tx_height
+        )
+-- denom match
+        select
+            tmj.denom as value,
+            'DENOM' as type
+        from tx_marker_join tmj , tx
+        where tmj.tx_hash_id = tx.id
```

Comment:
Just a nit here (and a waffly one at that).

At one point in my life, I decided it was better to explicitly define joins rather than listing all the tables in the from clause, and defining the join criteria in the where clause. I can't remember the reasons I decided that though. I'm thinking it was a performance thing, but a) it might not be an issue anymore, and b) there might be specific DB considerations that would make the SQL less portable. But at one point, I'm pretty sure doing as you have it caused the db to basically do a cross join first, then later filter out unwanted entries. But by defining the criteria and joins in the where clause, the db could skip the cross join and just create the desired results.

```suggestion
        from tx_marker_join tmj
        join tx on tmj.tx_hash_id = tx.id
```

What my memory is telling me is that, as the way you have it, the db does a cross join on the two tables first, then filters that with the criteria in the where clause. By defining it using joins (with the criteria in the from clause), the db starts with the first table and expands it with just what's needed based on the join criteria.